regression-tests/run-tests.sh

```diff
@@ -0,0 +1,306 @@
+    if [[ $test_name == "pure2"* ]]; then
+        descr="pure cpp2 code"
+        opt="-p"
+    fi
+    echo "    Testing $descr: $test_name.cpp2"
+
+    ########
+    # Run the translation test
+    echo "        Generating C++1 code"
+    ./"$cppfront_cmd" "$test_file" -o "$expected_src" $opt > "$expeced_output" 2>&1
+
+    failure=0
+    compiler_issue=0
+    ########
+    # Check the output
+    if [ -f "$expeced_output" ]; then
+        # The C++1 generation output has to be tracked
+        check_file "$expeced_output"
+        if [[ $untracked -eq 1 ]]; then
+            echo "            The C++1 generation output file is not tracked by git:"
+            echo "                $expeced_output"
+            failure=1
+        elif [[ -n "$diff_output" ]]; then
+                echo "            Non-matching std out/err:"
+                printf "\n$diff_output\n\n"
+                failure=1
+        fi
+    else
+        echo "            Missing expected output file treaded as failure"
+        failure=1
+    fi
+
+    ########
+    # Check the generated code
+    if [ -f "$expected_src" ]; then
+        # The file was generated, so it shoudl be tracked by git
+        check_file "$expected_src"
+        if [[ $untracked -eq 1 ]]; then
+            echo "            The generated file is not tracked by git:"
+            echo "                $expected_src"
+            failure=1
+        elif [[ -n "$diff_output" ]]; then
+            echo "            Non-matching generated src:"
+            printf "\n$diff_output\n\n"
+            failure=1
+        else
+            ########
+            # Compile and run the generated code in a sub-shell
+            expected_src_compil_out="$exec_out_dir/$generated_cpp_name.output"
+            expected_src_exec_out="$exec_out_dir/$generated_cpp_name.execution"
+            expected_files="$expected_results_dir/$test_name.files"
+
+            echo "        Compiling generated code"
+            
+            # For some tests the binary needs to be placed in "$exec_out_dir"
+            # For that reason the compilation is done directly in that dir
+            # The source is temporarily copied to avoid issues with bash paths in cl.exe
+            (cd $exec_out_dir; \
+             cp ../../$expected_src $generated_cpp_name;
+             $compiler_cmd"$test_bin" \
+                            $generated_cpp_name \
+                            > $generated_cpp_name.output 2>&1)
+            compilation_result=$?
+            rm $exec_out_dir/$generated_cpp_name
+
+            if [ -f "$expected_src_compil_out" ]; then
+                # Check for local compiler issues
+                if [[ $compilation_result -ne 0 ]]; then
+                    # Workaround an issue with MSVC mising std modules
+                    if cat $expected_src_compil_out | grep -q "error C1011"; then
+                        echo "            Skipping further checks due to missing std modules support"
```

Comment:
See also <https://github.com/modern-cmake/cppfront/issues/68>.

Reply:
Do you think I should include the issue link in a comment?

Reply:
No, this is just informational.